angularSTEP/src/app/profile-card/profile-card.component.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { async, ComponentFixture, TestBed, inject, tick, fakeAsync } from '@angular/core/testing';
+import { DebugElement, Component } from '@angular/core';
+
+import { By } from '@angular/platform-browser';
+
+import { ProfileCardComponent } from './profile-card.component';
+
+describe('ProfileCardComponent', () => {
+    let component: ProfileCardComponent;
+    let fixture: ComponentFixture<ProfileCardComponent>;
+    let de: DebugElement;
+
+    beforeEach(async(() =>{
+        TestBed.configureTestingModule({
+            declarations: [ ProfileCardComponent ],
+
+        })
+        .compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(ProfileCardComponent);
+        component = fixture.componentInstance;
+        de = fixture.debugElement;
```

Comment:
This is assigned but never used? Is it even needed?

Reply:
It doesn't have a use. I could never find a way to use it in this component